codeSnippets/snippets/docker/build.gradle

```diff
@@ -22,9 +20,7 @@ sourceSets {
 }
 
 repositories {
-    jcenter()
```

Comment:
The 'kotlinx-html-jvm' dependency still not moved from jcenter to mavenCentral https://github.com/Kotlin/kotlinx.html/issues/173. Removing jcenter fails a build now.

Reply:
Re-added but we need to remove this eventually.

Reply:
Sure, will do this under [https://youtrack.jetbrains.com/issue/KTOR-2183](KTOR-2183).